documentation/asciidoc/topics/proc_disabling_gossip_router_cross_site.adoc

```diff
@@ -0,0 +1,46 @@
+[id='disabling-gossip-router-cross-site_{context}']
+= Disabling local Gossip router and service
+
+[role="_abstract"]
+The {ispn_operator} starts a Gossip router on each site, but you only need a single Gossip router to manage traffic between the {brandname} cluster members.
+You can disable the additional Gossip routers to save resources.
+
+.Prerequisites
+
+* Have multiple working Gossip routers.
++
+You can have an external Gossip router that is not managed by the {ispn_operator}.
+
+.Procedure
+
+. Create an `Infinispan` CR for each {brandname} cluster.
+. Specify the name of the local site with `spec.service.sites.local.name`.
+. Set `false` as the value for the `spec.service.sites.local.discovery.launchGossipRouter`.
+. Connect to a remote site:
+.. For the remote site with Gossip router enabled,specify the `url` with the `spec.service.sites.locations.url`.
++
+[source,yaml,options="nowrap",subs=attributes+]
+----
+include::yaml/xsite_disable_gossip_router.yaml[]
+----
+.. For the remote site with Gossip router disabled, do not specify `spec.service.sites.locations.url`.
++
+[source,yaml,options="nowrap",subs=attributes+]
+----
+include::yaml/xsite_local_gossip_router.yaml[]
+----
```

Comment:
IMO splitting the procedure into two, with/without Gossip Router, like in Pedro's original commit is clearer.

Reply:
I tried one more time to show this as a single procedure. I used LON and NYC to make it clearer, but if it's confusing then I can introduce two procedures. np

Reply:
AFAICT the docs procedure is still instructing `spec.service.sites.local.discovery.launchGossipRouter` to be set to false for both sites.

Reply:
I think you might be checking an old diff?

Reply:
I just didn't read it carefully enough and missed that it was LON specifc :see_no_evil:

Reply:
And what do you think about the procedure as it is now? I updated the wording and I have a commit ready